packages/lib/src/components/PayByBankUS/PayByBankUS.tsx

```diff
@@ -0,0 +1,75 @@
+import { h } from 'preact';
+import { CoreProvider } from '../../core/Context/CoreProvider';
+import RedirectElement from '../Redirect';
+import RedirectButton from '../internal/RedirectButton';
+import { TxVariants } from '../tx-variants';
+import './PayByBankUS.scss';
+import getIssuerImageUrl from '../../utils/get-issuer-image';
+
+export default class PayByBankUS extends RedirectElement {
+    public static type = TxVariants.paybybank_AIS_DD;
+
+    /* TODO
+    DONE render component over PbB US
+    DONE - show correct disclaimer messages
+    TODO - upload translations  
+    TODO - fix logo
+    DONE - display brands
+    DONE - deduplicate css classes
+    TODO - check if needs clean/refactor
+    */
+
+    protected formatProps(props) {
+        return {
+            // paymentMethodBrands configuration
+            keepBrandsVisible: true,
+            showOtherInsteafOfNumber: true,
+            ...props
+        };
+    }
+
+    get displayName() {
+        return this.props.name || this.constructor['type'];
+    }
```

Comment:
Why do we need to override the `this.displayName` that is defined in the `UIElement`?  Can't we reuse it?

Reply:
FIxed